js/src/annotations/osd-svg-pin.js

```diff
@@ -82,18 +80,19 @@
         }
       }
     },
-    
-    onHover:function(activate,shape,hoverColor){
+
+    onHover:function(activate,shape,hoverWidth,hoverColor){
+      shape.strokeWidth = hoverWidth;
```

Comment:
The osd svg pin is rather service that operates with shapes it is not the shape itself